101-traffic-manager-virtual-machines/azuredeploy.json

```diff
@@ -0,0 +1,271 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
```

Comment:
the scenario is a basic one, the template unfortunately looks complicated because we have to create the VMs first.  we looked at using a separate template from the gallery to deploy the VMs but then we would have an ugly dependency on resource names used within that template so we opted to do it all in one place.